great-idea-website/css/index.css

```diff
@@ -63,4 +63,141 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.navigation-bar {
+	width: 850px;
+	top: 0;
+	border-bottom: 0px solid black;
+	position: fixed;
+	background-color: white;
+	width: 900px;
+	z-index: 5;
+}
+
+
+body {
+	margin: 0 auto;
+	width: 880px;
+	font-size: 16px;
+}
+
+header {
+	display: inline-block;
+	width: 100%;
+	position: relative;
+	background-color: white;
+	padding: 5px;
+}
+
+header nav a {
+	margin: 10px 28px;
+	font-size: 15px;
+	text-decoration: none;
+	color: grey;
+	display: inline-block;
+	background-color: white;
+}
+
+header nav a:first-of-type {
+	margin-left: 0;
+}
+
+header nav a:hover {
+	color: black;
+	font-size: 20px;
+	border-bottom: 2px dashed black;
+}
+
+
+.logo {
+	position: absolute;
+	right: 0;
+	padding-bottom: 4px;
+	background-color: white;
+}
+
+h1 {
+	font-size: 70px;
+}
+
+
+
+.first-section {
+	margin-top: 35px;
+	width: 880px;
+	height: 440px;
+	position: relative;
+	display: inline-block;
+	padding-top: 20px;
+}
+
+.first-section button {
+	border: 1px solid black;
+	background-color: white;
+	padding: 0 30px;
+	font-family: 'Titillium Web', sans-serif;
+	font-weight: 600;
+	font-size: 16px;
+	width: 170px;
+	height: 40px;
+	position: relative;
+}
+
+.left-side {
+	padding: 50px;
+	width: 400px;
+	text-align: center;
+	margin-top: 70px;
+	position: absolute;
+	display: inline-block;
+}
+
+.right-side {
+	padding: 50px 0;
+	right: 0;
+	width: 400px;
+	position: absolute;
+	display: inline-block;
+}
+
+.second-section {
+	padding: 30px 0 50px 0;
+	margin-top: 50px;
+	border-top: 1px solid black;
+}
+
+.section2 {
+	display: inline-block;
```

Comment:
I'm noticing that you're using a width of 400px a few times. This should prompt in future projects if it would be much better for you to create a container div that wraps these elements and just give that container a width of 400px, that way it can just pass it down to the children and you only have to set it once. Stay DRY.